From 64624f5584cadc7b964c503eae4eee3a4a310a68 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Sun, 10 Nov 2019 09:34:09 +0200
Subject: [PATCH] [SW-6739] lto: remove this flag from CFLAGS

It cause demos to fail at linkage since symbols from libsb.a are not
found e.g. sb_proxy_read
Failure is since compiler adds them lto prefix

Change-Id: I3053053c731abf7090faeb28598543b485926570
---

diff --git a/sdk/src/Makefile b/sdk/src/Makefile
index c3b40b0..a48cf61 100644
--- a/sdk/src/Makefile
+++ b/sdk/src/Makefile
@@ -43,7 +43,7 @@
 # Compiler Options
 CFLAGS += $(foreach dir,$(INCL_DIR),-I$(dir))
 
-CFLAGS += -Wall -W -O3 -flto -DELP_SOURCE
+CFLAGS += -Wall -W -O3 -DELP_SOURCE
 
 ifdef SB_RSA_FIXED
 CFLAGS += -DSB_RSA_FIXED
